apps/aechannel/src/aesc_fsm.erl

```diff
@@ -3343,7 +3346,7 @@ handle_common_event(E, Msg, M, #data{cur_statem_state = St} = D) ->
     handle_common_event_(E, Msg, St, M, D).
 
 handle_common_event_(timeout, St = T, St, _, D) ->
-    close({timeout, T}, D);
+    close({normal, T}, D);
```

Comment:
Why is a close by `timeout` tagged as `normal` now? Just seems semantically weird.

Reply:
I will elaborate it further so users receive more meaningful responses.